TransIT.DAL/Repositories/ImplementedRepositories/WorkTypeRepository.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using LinqKit;
+using Microsoft.EntityFrameworkCore;
+using TransIT.DAL.Models;
+using TransIT.DAL.Models.Entities;
+using TransIT.DAL.Repositories.InterfacesRepositories;
+
+namespace TransIT.DAL.Repositories.ImplementedRepositories
+{
+    public class WorkTypeRepository :BaseRepository<WorkType>,IWorkTypeRepository
+    {
+        public WorkTypeRepository(TransITDBContext context) : base(context)
+        {
+        }
+
+        public override Task<IQueryable<WorkType>> SearchExpressionAsync(IEnumerable<string> strArr)
```

Comment:
as I remember in pull request #50 SearchExpressionAsync was removed so please merge with latest dev and use the same approach as was done there